MAINTAINERS.md

```diff
@@ -0,0 +1,25 @@
+# Velero Maintainers
+
+[GOVERNANCE.md](https://github.com/vmware-tanzu/velero/blob/master/GOVERNANCE.md) describes governance guidelines and maintainer responsibilities.
+
+## Maintainers
+
+| Maintainer | GitHub ID | Affiliation |
+| --------------- | --------- | ----------- |
+| Steve Kriss | [skriss](https://github.com/skriss) | [VMware](https://www.github.com/vmware/) |
+| Carlisia Campos | [carlisia](https://github.com/carlisia) | [VMware](https://www.github.com/vmware/) |
+| Nolan Brubaker | [nrb](https://github.com/nrb) | [VMware](https://www.github.com/vmware/) |
+| Ashish Amarnath | [ashish-amarnath](https://github.com/ashish-amarnath) | [VMware](https://www.github.com/vmware/) |
+| Stephanie Bauman | [stephbman](https://github.com/stephbman) | [VMware](https://www.github.com/vmware/) |
+
+## Emeritus Maintainers
+
+## Velero Contributors & Stakeholders
+
+| Feature Area | Lead |
+| ----------------------------- | :---------------------: |
+| Technical Lead | Steve Kriss (skriss) |
+| Kubernetes CSI Liaison | Nolan Brubaker (nrb), Ashish Amarnath (ashish-amarnath) |
+| Deployment | Carlisia Campos (carlisia), Carlos Tadeu Panato Junior (cpanato) | 
+| Community Management | Jonas Rosland (jonasrosland) |
+| Product Management | Stephanie Bauman (stephbman) |
```

Comment:
Would it make sense to have additional feature areas like - release management, plugins, documentation/website?

Reply:
@kmova i am open to that once someone takes the lead for those areas. Meaning, if someone is willing to do that work and act as a lead in those areas, we can add that individual here too